pytest_fixtures/broker.py

```diff
@@ -0,0 +1,39 @@
+import pytest
+from broker.broker import VMBroker
+
+from robottelo.constants import BROKER_RHEL77
+from robottelo.hosts import ContentHost
+
+
+@pytest.fixture
+def rhel7_host():
+    """A function-level fixture that provides a host object based on the rhel7 nick"""
+    with VMBroker(nick='rhel7') as host:
+        yield host
+
+
+@pytest.fixture
+def rhel7_contenthost():
+    """A fixture that provides a content host object based on the rhel7 nick"""
+    with VMBroker(nick='rhel7', host_classes={'host': ContentHost}) as host:
+        yield host
+
+
+@pytest.fixture(scope="module")
+def module_rhel77_host():
```

Comment:
I would love to know the use cases of this fixture and `rhel7_host`. If both are doing the same then, you should be able to inherit module scoped `module_rhel77_host` in func scoped `rhel7_host` to override the scope.

Reply:
these are largely examples of fixture styles for us to discuss in this pr :)